tasks/setup-zypper.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: If choosing custom package set, ensure minimal cockpit is included
+  set_fact:
+    cockpit_packages: "{{ cockpit_packages + __cockpit_packages_minimal
+      if cockpit_packages not in __cockpit_package_types
+      else cockpit_packages }}"
+  when: cockpit_packages is defined
+
+- name: Ensure Cockpit Web Console packages are installed.
+  zypper:
```

Comment:
```suggestion
  community.general.zypper:
```
Also - change `meta/collection-requirements.yml` to look like this:
```yaml
# SPDX-License-Identifier: MIT
---
collections:
  - ansible.posix
  - community.general
  - fedora.linux_system_roles
```
That should make linters and ansible-plugin-scan happy

Reply:
Thank you for the suggestion, @richm . I have updated the meta/collection-requirements.yml as advised and made the necessary changes to the zypper reference